frontend/public/module/k8s/types.ts

```diff
@@ -753,19 +753,22 @@ export type MachineConfigPoolStatus = {
   updatedMachineCount: number;
   readyMachineCount: number;
   unavailableMachineCount: number;
-  conditions: MachineConfigPoolCondition[];
+  conditions?: MachineConfigPoolCondition[];
 };
 
 export type MachineConfigPoolSpec = {
   machineConfigSelector?: Selector;
   maxUnavailable?: number | string;
   nodeSelector?: Selector;
-  paused: boolean;
+  paused?: boolean;
```

Comment:
Is this optional when `spec` is there -- or is this a result of "if spec is optional, all children are optional"?

Reply:
My change is to cover the latter.  https://github.com/openshift/machine-config-operator/blob/4fbbead6d1a6e83d9736e75cdc359e10d5adfb84/pkg/apis/machineconfiguration.openshift.io/v1/types.go#L235-L262 doesn't specify any as required or optional.